src/Button/Button.scss

```diff
@@ -204,6 +204,20 @@
   }
 }
 
+.btn,
+.btn-primary,
+.btn-group-lg>.btn-primary.btn,
+.btn-group-sm>.btn-primary.btn,
+body .page-link {
+  .btn:focus::before,
```

Comment:
Is it OK, the `.btn` in the `.btn` class?

Reply:
Fixed, thanks